### PR TITLE
feat: add functionality to refresh the package config for an application

### LINKED ISSUE #27: Feature request: Expose an API method that allows refreshing eventListeners
# User Story

As a developer I have dynamically added fields in my FormApplication.
These fields don't get picked up by AIP. AIP won't know when to refresh
the Listeners for a given App/Sheet,  an API would be of great use.
 
As a developer I would like let AIP know when to refresh.

```javascript
// pseudocode
_addMyElements() {
 // add elements to the dom
 
 // let AIP know it should refresh the listeners for the module
 api = game.modules.get("autocomplete-inline-properties").API;
 api.refreshListeners(MODULE);
}
```

### PR DESCRIPTION
**Pull request by ghost91-**
Closes #27

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,3 +116,24 @@ Hooks.on("aipSetup", (packageConfig) => {
...
 });
 ```
+
+### Updating Event Listeners of an Application
+
+Here is an example of how updating the event listeners of an application could look like, for example after dynamically
+adding additional fields that match the selector of one of the field configs.
+
+```js
+function addAdditionalFields(activeEffectConfig) {
```
Comment (DanielBoettner) [inline CONTRIBUTING.md:126]: Some pseudo jsdoc could help to make it more clear that `activeEffectConfig` should be a reference to the FormApplication.

In my own implementation it was just "this" as a parameter for the `registerFieldConfigs` call.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -99,3 +116,24 @@ Hooks.on("aipSetup", (packageConfig) => {
...
+    /* code to add the additional fields to the app */
+
+    /* ... */
+
+    const {PACKAGE_CONFIG, registerFieldConfigs} =  game.modules.get("autocomplete-inline-properties").API;
+    const fieldConfigs = PACKAGE_CONFIG;
```
Comment (DanielBoettner) [inline CONTRIBUTING.md:132]: The semicolon should not be here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ghost91-) [reply CONTRIBUTING.md:130]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (ghost91-) [reply CONTRIBUTING.md]: obsolete now

**Pull request merged**